### PR TITLE
Update Web Audio API Status

### PR DESCRIPTION
**Pull request by hughrawlinson**
https://www.w3.org/blog/news/archives/9093

### CODE REVIEW

File: kumascript/macros/SpecData.json [hunk 1]
```diff
@@ -1632,7 +1632,7 @@
   "Web Audio API": {
     "name": "Web Audio API",
     "url": "https://webaudio.github.io/web-audio-api/",
```
Comment (escattone) [inline kumascript/macros/SpecData.json:1634]:
```suggestion
    "url": "https://www.w3.org/TR/webaudio/",
```

**Review by escattone — CHANGES_REQUESTED** [review]
Thanks @hughrawlinson! I'm thinking we should update the URL as well.

**Comment by hughrawlinson** [comment]
Sounds great, committed your suggestion

**Review by escattone — APPROVED** [review]
Thank you @hughrawlinson!

**Pull request merged**